moler/connection_observer.py

```diff
@@ -35,7 +35,7 @@ def __init__(self, connection=None):
         self._is_cancelled = False
         self._result = None
         self._exception = None
-        self.runner = ThreadPoolExecutorRunner()
+        self.runner = runner if runner else ThreadPoolExecutorRunner()
```

Comment:
I think so - it will allow any command/observer/event to work with any runner directly given.
Otherwise user needs to change .runner attribute if she/he decides to go with "non-default" runner.
That is also possible option to not complicate API - however, it is not visible (against Zen of python "Explicit is better than implicit.")

Moreover, in future, if runner parameter will be None we can get default-runner where "default" means "defult runner set up inside configuration" or "autodetected-runner" based on system capabilities.